examples/05_advanced/plot_ica_resting_state.py

```diff
@@ -45,10 +43,6 @@
                      mask_strategy='epi', standardize=True)
 data_masked = masker.fit_transform(func_filename)
 
-# Concatenate all the subjects
-# fmri_data = np.concatenate(data_masked, axis=1)
-fmri_data = data_masked
-
```

Comment:
And this is for maintaining compatibility with that?

Reply:
`fmri_data` is not called after this line AFAICT. Not sure how it would maintain compatibility...